README.md

```diff
@@ -1,38 +1,51 @@
-# Morpho Token
+# Morpho token
 
-This repository contains the Morpho protocol’s ERC20 token.
+This repository contains the Morpho protocol's ERC20 token.
 It is designed to be upgradable and support onchain delegation.
 Additionally, it ships a wrapper contract to simplify the migration of assets from the previous token contract to the new Morpho token contract.
 
-## Upgradability
+## Features
 
-The Morpho token complies with the [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+### Upgradeability
 
-## Delegation
+The Morpho token complies with [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+
+### Delegation
 
 The Morpho token supports onchain voting and voting power delegation.
 
+### Role-based permission
+
+The new Morpho token does not have role-based permission of functions.
+
+### Burning tokens
+
+In the legacy Morpho token, it was possible to transfer tokens to the zero address.
+This is no longer possible in the new Morpho token, but it's still possible for approved users to burn tokens by calling the `burn` function.
```

Comment:
Perhaps, a way to put emphasis on the "we're not breaking anything and this has the same features as before" we could write "In the legacy Morpho token contract, tokens could be burnt by transferring them to address zero. In this contract, tokens can be burnt directly with the `burn` function.

Reply:
I don't think we should do it because transferring to the zero address is not the same as burning tokens (total supply doesn't change in the first case)

Reply:
Maybe I should remove the word "still", it could be misleading in this way

Reply:
Doing the change to remove "still", but not changing otherwise because we don't want to convey that it did not have burning feature before

Reply:
I see you're point 🤔. It's just that the sub-title here is named "Burning tokens" and the transfer to zero address in mentioned, this could be a point of misunderstanding as well, don't you think?

Reply:
Hmm I think it's fine, the title can't convey everything. What would be the misunderstanding with the new phrasing ?

Reply:
It could be understood as if burning tokens and transferring to zero is the same, which are not according to your observation. It's clear enough for me, but maybe it's worth getting the legal point of view from someone, just to make sure that we're not getting into trouble.